generator/generator.yml

```diff
@@ -195,3 +195,26 @@ modules:
       new_index: ifDescr
     - old_index: laIndex
       new_index: laNames
+
+# Ubiquiti / AirMax
+#
+# https://community.ubnt.com/ubnt/attachments/ubnt/airMAX_FAQ/84/3/ubnt-mib.04.06.2017.zip
+#
+  ubiquiti_airmax:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
+      - 1.3.6.1.4.1.41112.1.4
```

Comment:
We usually put the object name in a comment

Reply:
Ahh, whups, forgot that one.